tools/theme/src/Theme.ts

```diff
@@ -158,40 +175,56 @@ export class Theme extends HTMLElement implements ThemeKindProvider {
 
     public override shadowRoot!: ShadowRootWithAdoptedStyleSheets;
 
-    private _theme: ThemeVariant | '' = 'spectrum';
-
+    private _system: SystemVariant | '' = 'spectrum';
     /**
-     * The Spectrum theme that is applied to the content scoped to this `sp-theme` element.
+     * The Spectrum system that is applied to the content scoped to this `sp-theme` element.
      *
      * A value is requried.
-     * @type {"spectrum" | "express" | ""}
+     * @type {"spectrum" | "express" }
      * @attr
      */
-    get theme(): ThemeVariant | '' {
-        const themeFragments = Theme.themeFragmentsByKind.get('theme');
+    get system(): SystemVariant | '' {
+        const themeFragments =
+            Theme.themeFragmentsByKind.get('system') ||
+            Theme.themeFragmentsByKind.get('theme');
         const { name } =
             (themeFragments && themeFragments.get('default')) || {};
-        return this._theme || (name as ThemeVariant) || '';
+        return this._system || (name as SystemVariant) || '';
     }
 
-    set theme(newValue: ThemeVariant | '') {
-        if (newValue === this._theme) return;
-        const theme =
-            !!newValue && ThemeVariantValues.includes(newValue)
+    set system(newValue: SystemVariant | '') {
+        if (newValue === this._system) return;
+        const system =
+            !!newValue && SystemVariantValues.includes(newValue)
                 ? newValue
-                : this.theme;
-        if (theme !== this._theme) {
-            this._theme = theme;
+                : this.system;
+        if (system !== this._system) {
+            this._system = system;
             this.requestUpdate();
         }
-        if (theme) {
-            this.setAttribute('theme', theme);
+        if (system) {
+            this.setAttribute('system', system);
             /* c8 ignore next 3 */
-        } else {
-            this.removeAttribute('theme');
         }
     }
 
+    /*
+     * @deprecated The `theme` attribute has been deprecated in favor of the `system` attribute.
+     */
+    get theme(): SystemVariant | '' {
+        if (!this.system) {
+            this.removeAttribute('system');
+        }
+        return this.system;
+    }
+
+    /*
+     * @deprecated The `theme` attribute has been deprecated in favor of the `system` attribute.
+     */
+    set theme(newValue: SystemVariant | '') {
+        this.system = newValue;
```

Comment:
While this protects for a consumer setting `theme` only, over and over again, it also means that if a consumer uses both `system` AND `theme` and uses `theme` _last_ that `theme` will always win.... Is there a way to have `system` win if it is provided by the consumer, always?

Reply:
I have added this check here to make sure if a consumer sets both 'system` and 'theme`, 'system` will take precedence and theme will not be set.
```js
if (this.system !== '') {
        // System is provided, so do not set theme
        return;
    }

    this.system = newValue;
```